contrib/cirrus/setup_environment.sh

```diff
@@ -61,11 +61,15 @@ then
         fedora-29)
             CON_SEL="https://kojipkgs.fedoraproject.org/packages/container-selinux/2.100/1.git3b78187.fc29/noarch/container-selinux-2.100-1.git3b78187.fc29.noarch.rpm"
             echo ">>>>> OVERRIDING container-selinux WITH $CON_SEL <<<<<"
-            dnf -y install $CON_SEL
+            dnf -y install $CON_SEL https://kojipkgs.fedoraproject.org//packages/selinux-policy/3.14.2/56.fc29/noarch/selinux-policy-3.14.2-56.fc29.noarch.rpm https://kojipkgs.fedoraproject.org//packages/selinux-policy/3.14.2/56.fc29/noarch/selinux-policy-devel-3.14.2-56.fc29.noarch.rpm https://kojipkgs.fedoraproject.org//packages/selinux-policy/3.14.2/56.fc29/noarch/selinux-policy-targeted-3.14.2-56.fc29.noarch.rpm
```

Comment:
I think this is released now, and newer selinux-policy is available, so should just install latest.